src/services/SearchService.ts

```diff
@@ -60,60 +59,6 @@ async function search(entity: SearchRequestPayload): Promise<SearchResponseEleme
   }
 }
 
-type SearchFieldType = {
```

Comment:
moved all this into TrackingService, SearchService was to be for more generic functions